packages/office-ui-fabric-react/src/components/Autofill/Autofill.tsx

```diff
@@ -267,28 +270,28 @@ export class Autofill extends BaseComponent<IAutofillProps, IAutofillState> impl
     }
   }
 
-  private _notifyInputChange(newValue: string): void {
+  private _notifyInputChange(newValue: string, wasComposing: boolean): void {
     if (this.props.onInputValueChange) {
-      this.props.onInputValueChange(newValue);
+      this.props.onInputValueChange(newValue, wasComposing);
```

Comment:
Could we name this variable just `composing` I think it clarifies what it means and then consuming functions can choose to consume the value when `composing` is false.

Reply:
sure. I don't generally like using non (is/was/should) names for booleans, but since the is/was is ambiguous I'm fine with it